Doc-11534-PM-2.8-Edits-DH  Partner Manager 2.8 edits (#141)
mulesoft/docs-partner-manager: modules/ROOT/Edits/modify-partner-settings.adoc

= Modify a Partner's Settings

> You can modify a partner's settings including contact details, identifiers, digital certificates, acknowledgements, control numbers, and more.

Reviewer: ```suggestion
You can modify partner settings such as contact details, identifiers, digital certificates, acknowledgments, and control numbers.
```

Reviewer: Be sure that the subtopics for each follow the order in which they are introduced here.

To modify a partner's settings:

. In Partner Manager, select *Partners* in the navigation menu.

> . Click the partner.

Reviewer: Compare with earlier wordings:

- Select the partner for whom to configure the send settings.
- Select the partner to associate with the settings.
- Select the partner for whom you want to configure the settings.

I know it is tedious, but if you have time, review the words used for repeated instructions and try to standardize them. I wouldn't even notice if I did not have so much of this text at one time...and probably the audience will not. But it is still a good goal to aspire to. :-)

> . Change any of the following settings:

Reviewer: You typically do not want a table within a task...especially a table that contains substeps.
